### ISSUE TITLE
Lower score for matching on the middle

### ISSUE THREAD

**Issue by taooceros** [Flow-Launcher/Flow.Launcher#328]
![image](https://user-images.githubusercontent.com/45326534/107003837-4353cd00-6742-11eb-8a93-bf11425ea3e3.png)
I think we should lower the score for this matching.

**Comment by jjw24** [comment]
this is a match from fuzzy not acronym i think

**Comment by taooceros** [comment]
Yeah it is from fuzzy. The score of fuzzy is calculated based on the nearest space.
